__tests__/app.test.js

```diff
@@ -28,7 +28,7 @@ describe(`GET /api/topics`, () => {
     })
 });
 
-describe(`GET /api/articles/:article_id`, () => {
+describe.only(`GET /api/articles/:article_id`, () => {
```

Comment:
You must remove any .only, xtest, xit before submitting a pull request. How can we be sure these changes do not break earlier code?

Reply:
Gah! I literally spotted it and re-pushed just as you were writing this... On it!